### PR TITLE
added all websites and travis

### PR DESCRIPTION
**Pull request by mosaadm**
Fixes #\<GitHub-issue-number\>.

Make sure all boxes are checked (add x inside the brackets) when you submit your contribution, remove this sentence before doing so.

- [ ] This PR is compliant with the contributing guidelines (if not, please describe why): code is fully documented and if possible a .md file is made.
- [ ] The PR text includes a **detailed explanation** (more than 50 chars)
- [ ] I have thoroughly tested my contribution.

\<Description of and rational behind this PR\>

**In case of a PR concerning the Mooltipass extension:**  
The following test procedure should be done, in the following order.  
  
1) Recall functionality test (Mooltipass unlocked):  
- visit a website you have credentials for  
- make sure you get prompted by your mooltipass  
- accept the credentials sending request  
- make sure you are logged in  
  
2) Storage functionality test (Mooltipass unlocked):  
- visit a website you don't have credentials for  
- fill the username and password field, submit  
- make sure the mooltipass prompts you for password storage **once**  
- accept the credentials storage request  
- run test 1) to make sure credentials are correctly stored  
  
3) Cancel functionality test (Mooltipass unlocked):  
- visit a website you have credentials for  
- make sure you get prompted by your mooltipass   
- do not accept the credentials sending request, close the tab  
- make sure the prompt gets removed on the mooltipass  
  
4) Credentials requests queue test (Mooltipass **locked**)  
- visit a website you have credentials for  
- unlock your mooltipass  
- make sure you get prompted by your mooltipass   
  
5) Credentials requests queue test (Mooltipass unlocked)  
- visit a website A you have credentials for  
- make sure you get prompted by your mooltipass   
- do not accept the credentials sending request  
- visit a website B you have credentials for  
- close the tab containing the website A  
- make sure the first prompt is cancelled on the mooltipass  
- make sure another prompt for website B is displayed on the mooltipass

### CODE REVIEW

File: chrome_extension/updateCleanVersion.sh [hunk 1]
```diff
@@ -105,7 +105,7 @@ function _build()
     )
 
     [ "$BUILD_FIREFOX" == 1 ] &&_build_firefox_xpi "${zip_file}" "${BUILD_METADATA[${FIREFOX_TARGET}]}"
-    [ "$BUILD_CHROMIUM" == 1 ] &&_build_chromium_crx "${zip_file}" "${BUILD_METADATA[${CHROMIUM_TARGET}]}"
+    [ "$BUILD_CHROMIUM" == 1 ] &&_build_chromium_crx "${zip_file}" "${BUILD_METADATA[${CHROMIUM_TARGET}]}"|| true
```
Comment (limpkin) [inline chrome_extension/updateCleanVersion.sh]: why's that?

File: chrome_extension/vendor/mooltipass/device.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ mooltipass.device = mooltipass.device || {};
 /**
  * Mooltipass emulation vars
  */
-mooltipass.device.emulation_mode = false
+mooltipass.device.emulation_mode = true
```
Comment (limpkin) [inline chrome_extension/vendor/mooltipass/device.js]: set to false

File: chrome_extension/test.sh [hunk 3]
```diff
@@ -0,0 +1 @@
+echo Hi
```
Comment (limpkin) [inline chrome_extension/test.sh]: is that needed?

Comment (mosaadm) [inline chrome_extension/test.sh]: this can be removed

File: chrome_extension/updateCleanVersion.sh (hunk 1 above)
Comment (mosaadm) [inline chrome_extension/updateCleanVersion.sh]: this is so that it returns true and travis continues the build

Comment (limpkin) [inline chrome_extension/updateCleanVersion.sh]: right... I'm guessing that you may modify the .sh to set the boolean to true then :)

File: sauce_labs_tests/src/test/java/mooltipass/automatedTest/config/WebDriverFactory.java [hunk 4]
```diff
@@ -60,36 +60,46 @@ public static WebDriver get() {
...
 	}
 	
 	//for local testing on firefox
-	private static WebDriver firefox(){
+	private static WebDriver firefox(String extension){
 		System.setProperty("webdriver.gecko.driver","src/test/resources/geckodriver.exe");
 		FirefoxOptions options = new FirefoxOptions();
 		FirefoxProfile profile = new FirefoxProfile();
-		profile.addExtension(new File("/path_to_extension"));
+		profile.addExtension(new File("C:\\Users\\mohamed\\Downloads\\mooltipass_extension-1.1.81-an+fx-windows.xpi"));
```
Comment (limpkin) [inline sauce_labs_tests/src/test/java/mooltipass/automatedTest/config/WebDriverFactory.java]: is that running on sauce labs? how is that working if your user name is there?

**Pull request closed without merging**